Makefile

```diff
@@ -59,8 +62,8 @@ $(DEB): tmp/build/toxiproxy-server-linux-amd64 tmp/build/toxiproxy-client-linux-
 		--description "TCP proxy to simulate network and system conditions" \
 		--url "https://github.com/Shopify/toxiproxy" \
 		$(word 1,$^)=/usr/bin/toxiproxy \
```

Comment:
Not sure if we should leave this or not, I don't really have much of an opinion on it